docs/builders/iso.mdx

```diff
@@ -355,11 +401,7 @@ or responding to pattern `/dev/.+`. Example:
         * hdiutil (normally found in macOS)
         * oscdimg (normally found in Windows as part of the Windows ADK)
 
-
-    - `cd_content` (map[string]string) - Key/Values to add to the CD. The keys represent the paths, and the values
-      contents. It can be used alongside `cd_files`, which is useful to add large
-      files without loading them into memory. If any paths are specified by both,
-      the contents in `cd_content` will take precedence.
```

Comment:
This is a drive-by fix for what I hope will rectify the following situation on the website:

<img width="733" alt="image" src="https://github.com/hashicorp/packer-plugin-proxmox/assets/6570684/1cf7215f-963b-4a89-9310-0b6618a3464e">

Reply:
FYI, PR #200 fixes this problem, so I'll merge this and you can leave this change out of yours after that!

Reply:
Thank you, that's really useful! 🎉